pulumiyaml/program.go

```diff
@@ -0,0 +1,118 @@
+package pulumiyaml
+
+import "fmt"
+
+type Program struct {
+	// Defaults to "test"
+	Name string `yaml:"name,omitempty"`
+	// Defaults to "yaml"
+	Runtime     string              `yaml:"runtime,omitempty"`
```

Comment:
This makes it clearer to me which field the default applies to.
```suggestion
	Name string `yaml:"name,omitempty"` // Defaults to "test"
	Runtime     string              `yaml:"runtime,omitempty"` // Defaults to "yaml"
```

Reply:
This will all go away. Given these are required fields, we'll just default them in the `NewInlinePulumiTest` and `WriteProject` methods.